server/commandManager.py

```diff
@@ -0,0 +1,154 @@
+# Manager for sending and verifying route commands
+import logging
+
+from qr import QrTypes, QrHandler
+from telemetryHandler import TelemetryHandler
+from route import RouteTypes
+from waypoint import Waypoint
+from flaskClientSocket import FlaskClientSocket
+from loggingHandler import setup_logging
+
+setup_logging()
+
+# Boolean to email flight plan or send flight plan to drone to execute
+TASK_2_EMAIL_DAY = True
+
+# Weight of drone used to filter Task 2 routes with weight limits
+VEHICLE_WEIGHT = 0
+
+
+class CommandManager:
+
+    def __init__(self,
+                 qr_handler: QrHandler,
+                 telemetry_handler: TelemetryHandler):
+        self.qr_handler = qr_handler
+        self.telemetry_handler = telemetry_handler
+        self.route = []
+        self.updated_route = []
+        self.current_command = None
+        self.backup_command = None
+        self.sent_command = None
+        self.sent_message = ""
+        self.socket = FlaskClientSocket(1, 8080)
+
+        # TODO: Without Flight/CommunicationRouter, socket connection fails
+        # Requires button on website on whether to flight is initialized or not
+        # self.socket.connect()
+
+    def connect_to_socket(self):
+        """Start socket connection
+
+        :return: True if successful connection, else False
+        """
+        return self.socket.connect()
+
+    def send_command(self, message):
+        """Send message to Flight
```

Comment:
Would this be to the communication router instead?

Reply:
Technically yes, with intended target Flight